lib/framework/action.dart

```diff
@@ -445,6 +445,18 @@ class CopyToClipboardAction extends EnsembleAction {
   String? value;
```

Comment:
what if it's an integer? use dynamic here

Reply:
Changed it to dynamic